walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -24,6 +24,11 @@
   border-style: solid;
 }
 
+.font {
```

Comment:
You probably want to have a more descriptive class name since just "font" doesn't tell you a whole lot about what this class does

Reply:
font is also possibly a bit confusing, since font is a valid css rule name.

I'll have to add talking about different naming conventions to the list of things to discuss Friday.

Reply:
Will change it right away, thank you!